setup.py

```diff
@@ -115,7 +115,6 @@ def get_requirements(requirements_file):
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
```

Comment:
Out of curiosity, why are you removing this line?

Reply:
Ah, feanil pointed out in another PR that line should be gone, but I had merged it, so i'm just adding in the change here :)